### PR TITLE
Homepage milestone1

### PR DESCRIPTION
**Pull request by hoangviet1993**
This PR sets to setup the basic structure for the homepage.

1)I broke out the lengthy sentences into different paragraph per the mocks.

2)Added the landing page image and added box-border for responsiveness.

3)Restructure index.html and home.html using material design tags

### CODE REVIEW

File: pages/home/home.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      <p>
+        By making a donation to The Oppia Foundation, you are helping to change millions of students’ lives. Together, we can revolutionize the way students around the world use the internet to learn.
+      </p>
+    </md-content>
+  </md-content>
+</section>
+
+<style>
+
+  .welcome_image {
```
Comment (seanlip) [inline pages/home/home.html]: .home-page-welcome-image

(typically, use kebab-case for CSS, and let's prefix it with the page that the CSS is on unless you want to share the CSS with other pages.)

File: pages/home/home.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    border: 0;
```
Comment (seanlip) [inline pages/home/home.html]: Alphabetize: border then box-sizing then position then width.

File: pages/home/home.html (hunk 1 above)
Comment (hoangviet1993) [reply pages/home/home.html]: Fixed :)

File: pages/home/home.html (hunk 2 above)
Comment (hoangviet1993) [reply pages/home/home.html]: Fixed!

File: pages/home/home.html [hunk 3]
```diff
@@ -0,0 +1,255 @@
+<section id="main-content">
+    <md-card layout-fill>
```
Comment (seanlip) [inline pages/home/home.html]: Deindent this and all following lines by 2. Indentation in HTML should be by 2 spaces.

**Comment by hoangviet1993** [comment]
I think this is a good breaking point for this PR.
Matches 60% of the mock (imo). I need some pointer on UX design / responsiveness soon should I want a higher % matches.
Working on overlaying the welcome image with just HTML/CSS.
Current home page:
home_page_07222018

File: pages/home/home.html [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+            <div>
+          </md-card-actions>
+        </md-content>
+      </md-card-content>
+    </md-card>
+</section>
+
+<style>
+
+  .white-background {
```
Comment (seanlip) [inline pages/home/home.html]: Start every oppia-related CSS class in this file with .home-page- ; follow a similar convention for other pages.

That also helps us differentiate from CSS local to this page vs general CSS or CSS from third-parties.

File: pages/home/home.html [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+  
+  .home-page-partner-image {
+    border: 0;
+    display:block;
+    max-height: 600px;
+    max-width: 600px;
+    margin:auto;
+  }
+
+  .green-text{
```
Comment (seanlip) [inline pages/home/home.html]: Add space before {, here and below.

File: static/css/devices.min.css [hunk 6]
```diff
@@ -0,0 +1 @@
...ice-frame{background:#0d0d0d;border-radius:40px;box-shadow:inset 0 0 2px 2px #adb0b3,inset 0 0 0 6px #e2e3e4,inset 0 0 0 8px #e2e3e4;height:234px;padding:32px;position:relative;width:200px}.device-apple-watch .device-frame::after{border-radius:30px;box-shadow:inset 0 0 25px 0 rgba(255,255,255,.75);content:"";height:216px;left:9px;position:absolute;top:9px;width:182px}.device-apple-watch .device-content{border:2px solid #121212;border-radius:2px;height:170px;width:136px}.device-apple-watch .device-btns{background:#e2e3e4;border-left:2px solid #adb0b3;border-radius:8px 4px 4px 8px/20px 4px 4px 20px;box-shadow:inset 0 0 2px 2px #adb0b3;height:44px;position:absolute;right:-10px;top:52px;width:16px;z-index:9}.device-apple-watch .device-btns::after{background:#e2e3e4;border-radius:4px 2px 2px 4px/10px 2px 2px 10px;box-shadow:inset 0 0 1px 2px #adb0b3;content:"";height:66px;position:absolute;right:6px;top:68px;width:8px}.device-apple-watch .device-btns::before{background:#adb0b3;box-shadow:0 -16px #adb0b3,0 -12px #adb0b3,0 -8px #adb0b3,0 -4px #adb0b3,0 4px #adb0b3,0 8px #adb0b3,0 12px #adb0b3,0 16px #adb0b3;content:"";height:2px;margin-top:-1px;position:absolute;right:0;top:50%;width:6px}
```
Comment (seanlip) [inline static/css/devices.min.css]: Better not to ship third-party stuff directly. Have that be part of the start script, create a third-party folder, etc.

**Comment by seanlip** [comment]
(Also, visually, it's looking good!)

File: pages/home/home.html (hunk 3 above)
Comment (hoangviet1993) [reply pages/home/home.html]: Done!

File: pages/home/home.html (hunk 4 above)
Comment (hoangviet1993) [reply pages/home/home.html]: All css classes in home.html should now be exclusive to home page.
Moving common css classes to stylesheet.css.

File: pages/home/home.html (hunk 5 above)
Comment (hoangviet1993) [reply pages/home/home.html]: Done!

File: static/css/devices.min.css (hunk 6 above)
Comment (hoangviet1993) [reply static/css/devices.min.css]: Move third party css sheet to /third_party/css

File: pages/home/home.html (hunk 3 above)
Comment (seanlip) [reply pages/home/home.html]: Not done?

**Comment by hoangviet1993** [comment]
One more check-in for this PR. @seanlip PTAL, thanks! 👍 
Mobile view:
home_page_mobile_view
Regular view:
home_page_reg_view

File: pages/home/home.html (hunk 3 above)
Comment (hoangviet1993) [reply pages/home/home.html]: Didn't actually see the <md-card> tags. My bad there.

**Comment by hoangviet1993** [comment]
I have added the scripts for installing third party dependdencies and removed the added devices.min.css!

File: pages/home/home.html [hunk 7]
```diff
@@ -0,0 +1,197 @@
+<section id="main-content">
+  <md-card layout-fill>
+    <img ng-src="https://placekitten.com/3000/1200" class="home-page-welcome-image md-card-image">
```
Comment (seanlip) [inline pages/home/home.html]: LOL

File: scripts/install_third_party.sh [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+# limitations under the License.
+
+set -e
+source $(dirname $0)/setup.sh || exit 1
+
+# Download and install required JS and zip files.
+echo "Installing third-party JS libraries and zip files."
+
+if [ ! -f $THIRD_PARTY_DIR/css/devices.min.css ]; then
+    echo "Downloading devices.css"
```
Comment (seanlip) [inline scripts/install_third_party.sh]: Use 2-space indentation

File: pages/home/home.html [hunk 9]
```diff
@@ -0,0 +1,197 @@
...
+          By uniting a global community of volunteers and educators, Oppia’s lessons are continually translated into new languages and adapted for different cultures.
+        </p>
+      </md-content>
+    </md-card-content>
+  </md-card>
+
+  <md-card layout-fill class="oppia-foundation-green-background oppia-foundation-vertical-padding">
+    <md-card-content layout="row" layout-align="center">
+      <md-content layout="column" class="oppia-foundation-white-text oppia-foundation-green-background home-page-history-text-box">
+          <center>
```
Comment (seanlip) [inline pages/home/home.html]: Use 2-space indentation

File: pages/home/home.html [hunk 10]
```diff
@@ -0,0 +1,197 @@
...
+              Since launching Oppia.org in 2015, over 800,000 learners in more than 20 countries have used our platform to explore, grow, and learn.
```
Comment (seanlip) [inline pages/home/home.html]: Use 2-space indentation

File: pages/home/home.html [hunk 11]
```diff
@@ -0,0 +1,197 @@
...
+      <md-content layout="column" class="oppia-foundation-black-text oppia-foundation-gray-background oppia-foundation-horizontal-padding" layout-align="center none">
+        <h3>Partner with us</h3>
+        <p>
+          We are looking for organizations that want to join us on our journey to transform how students learn online.
+        </p>
+        <p>
+          With your help, we aim to improve our platform by conducting randomly controlled trials, fund a dedicated team of lesson creators, and manage the costs of scaling the Oppia project to serve learners across the world.
+        </p>
+        <md-card-actions layout-xs="column" layout-align="center center">
+            <md-button class="oppia-foundation-black-text md-raised md-no-ink oppia-foundation-white-background oppia-foundation-gold-border-button md-raised">
```
Comment (seanlip) [inline pages/home/home.html]: Use 2-space indentation, here and below

**Comment by hoangviet1993** [comment]
Final (hopefully) update: Completed text overlay of welcome image. Awaiting high-res images/assets to replace screenshots.
Regular view:
home_page_reg_view
Mobile view: (partial screenshot)
screen shot 2018-07-23 at 12 46 55 am

File: pages/home/home.html (hunk 7 above)
Comment (hoangviet1993) [reply pages/home/home.html]: What can I say...I love kittens?

File: scripts/install_third_party.sh (hunk 8 above)
Comment (hoangviet1993) [reply scripts/install_third_party.sh]: Fixed!

File: pages/home/home.html (hunk 9 above)
Comment (hoangviet1993) [reply pages/home/home.html]: Done.And ditto below. I think I did a lot of copy-and-pasting thus messing up the indentations..I should be more careful!

File: pages/home/home.html [hunk 12]
```diff
@@ -0,0 +1,225 @@
...
+  }
+
+  .home-page-donation-text-box {
+    max-width: 500px;
+  }
+
+  .home-page-history-text-box {
+    max-width: 500px;
+  }
+</style>
```
Comment (seanlip) [inline pages/home/home.html]: End with EOL

Comment (hoangviet1993) [reply pages/home/home.html]: Done! Thanks!

**Pull request merged**